appeals/api/src/server/utils/compare-strings.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @param {string} stringA
+ * @param {string} stringB
+ * @returns {boolean}
+ */
+const compareStrings = (stringA, stringB) =>
```

Comment:
Just a suggestion, wouldn't be clearer with a function name that indicates testing for equality?

Reply:
Yes, good point, I've updated that to `checkStringsMatch`